src/App/Documentation/components/ProgressTracker/index.js

```diff
@@ -0,0 +1,120 @@
+import React from "react";
+import PrismCode from "react-prism";
+
+import ComponentPreview from "../../utils/ComponentPreview";
+import DocToc from "../../utils/DocToc";
+
+import ProgressTrackerComponent from "../../../components/ProgressTracker";
```

Comment:
I would suggest looking at something like https://github.com/entwicklerstube/babel-plugin-root-import to avoid having to write `import ProgressTrackerComponent from "../../../components/ProgressTracker";` and instead write `import ProgressTrackerComponent from '@/components/ProgressTracker'`